src/libraries/Microsoft.PowerFx.Json/Functions/JsonFunctionImpl.cs

```diff
@@ -254,6 +258,17 @@ public void Visit(TableValue tableValue)
                 {
                     _writer.WriteStartArray();
 
+                    var isSingleColumnValueTable = false;
+                    if (_unwrapValueTables)
+                    {
+                        var fieldTypes = tableValue.Type.GetFieldTypes();
+                        var firstField = fieldTypes.FirstOrDefault();
+                        if (firstField != null && !fieldTypes.Skip(1).Any() && firstField.Name.Value == "Value")
```

Comment:
>"Value" [](http://example.com/codeflow?start=104&length=7)

We don't have any existing constant for it?

Reply:
Updating with constant